content/about/about_people.rst

```diff
@@ -288,8 +288,10 @@ zimmerry
 
 Ryan joined the OSL in June 2020 and is studying Computer Science with a focus on cybersecurity. 
 He first learned of the OSL through Khuong and other OSL alumni during a Cyber Defense Competition.
-Ryan is the lab manager for the OSU Security Club (OSUSEC), and hopes to be able to use the knowledge he gains at the OSL to improve the club's systems.
-He also loves playing videogames, learning how to build exciting things, and participating in cometitions with the Security Club. 
+Ryan is the lab manager for the OSU Security Club (OSUSEC), and hopes to be able to use the knowledge he gains 
+at the OSL to improve the club's systems.
+He also loves playing video games, learning how to build exciting things, and participating in competitions with the 
+Security Club. 
```

Comment:
This still doesn't look like it's wrapped properly. I know in vim you can select the text and then run a command to force automatically wrapping it based on the text width setting. Can you please see if there's something similar with VS Code and adjust this to 120char?

Reply:
I fixed the wrapping, but I dont see an extension to do it in vscode. I can wrap it from my view but it doesnt wrap how it was displayed in the docs